content/en-us/reference/engine/classes/Actor.yaml

```diff
@@ -24,7 +24,7 @@ methods:
       Binds a Luau callback to a message with the specified topic.
     description: |
       This method is used to bind a Luau callback to a message with the
-      specified topic. When a message is sent (using `Actor/SendMessage`) to the
+      specified topic. When a message is sent (using `Class.Actor:SendMessage()|SendMessage`) to the
```

Comment:
```suggestion
      specified topic. When a message is sent (using `Class.Actor:SendMessage()|SendMessage()`) to the
```

Reply:
Just the smallest edit on these, adding the trailing `()` so that these links still appear as methods to a reader, and not mistaken with a property or event.